Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN apk --update upgrade && \
     update-ca-certificates && \
     rm -rf /var/cache/apk/*
 
+# healthchecks: requires curl if not present in container
+# RUN apk --update --no-cache add curl
+HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 CMD curl --fail http://localhost:8080/status || exit 1
```

Comment:
Timeout seems to be too long. I would suggest to use maybe 3 to 5 seconds. It should be at least shorter than the interval...

Reply:
all scanners now have the same 5s interval